openstf/model/objective.py

```diff
@@ -78,49 +149,49 @@ def __call__(self, trial: optuna.trial.FrozenTrial) -> float:
         # Configure evals for early stopping
         eval_set = [(train_x, train_y), (valid_x, valid_y)]
 
-        model = ModelCreator.create_model(self.model_type)
-
-        params = self.get_params(trial)
+        # get the parameters used in this trial
+        param = self.get_trial_parameters(self.model_params, trial)
```

Comment:
Like the previous comment, something like `trial_params` or `trial_parameters` would be more clear.

Reply:
Like Frank also mentioned, if this is a method only for internal use you should prefix it with `_` and in that case you don't need to pass the `self.model_params` since the method will have access to self anyway.